_posts/release/2025-05-01-accumulo-2.1.4.md

```diff
@@ -0,0 +1,144 @@
+  system property (see #20084 for details) to suppress the exception.
+
+* {% ghi 5073 %} {% ghi 5034 %} Added `accumulo check-accumulo-properties` command that can be
+  run before an instance is initialized to check the properties file.
+
+* {% ghi 5193 %} Added `accumulo admin signalShutdown` command to signal the server
+  process to initiate a graceful shutdown. Compactors will finish the major compaction
+  that they are currently working on, then will exit. Scan Servers will return a busy
+  signal to all clients for new scan requests, and will shutdown when all in-progress
+  scans have closed. Tablet Servers will signal the Manager that they are shutting down,
+  which should prevent assignment of tablets to that server, then they will unload all
+  tablets and shut down. The Monitor, Manager, GarbageCollector, and CompactionCoordinator
+  will shut down also. The last step in the shutdown process for all servers is to remove
+  their lock in ZooKeeper.
+
+* {% ghi 5438 %} Added `accumulo upgrade --prepare` command which should be used after
+  shutting down an instance in preparation for an upgrade. This will check that no Fate
+  transactions exist, delete any ZooKeeper locks for server processes, and prohibit any
+  server processes from being started.
+
+* {% ghi 4898 %} New way to compute bulk load plans (TODO: Needs text)
+
+* {% ghi 5169 %} {% ghi 5170 %} The TabletServer will halt itself when a walog write or
+  minc failure occurs and the TabletServer lock is not held in ZooKeeper.
+
+* {% ghi 5145 %} {% ghi 5132 %} The Manager and TabletServer processes now have a background
+  thread that can be enabled via the property `general.server.lock.verification.interval` to
+  validate that the process is holding the lock in ZooKeeper. The existing mechanism of relying
+  on a Watcher to notify the process that the lock has been lost can be a problem due to the
+  fact that there is a single thread in the ZooKeeper client that fires Watcher events. If the
+  thread is currently waiting on a hung Watcher, then subsequent Watcher events will not fire
+  and could leave the server running without a lock.
+
+* {% ghi 5174 %} Improvements to the `accumulo-cluster` and `accumulo-service` scripts have been
+  backported from the main branch. This includes syntax changes.
+
+* {% ghi 5384 %} {% ghi 5386 %} Fixed an issue with multiple threads performing AES decryption leading to corrupt data.
+
+### Configurable Improvements
+
+Many notable improvements have been added that include a means for users to configure them via new
+or existing properties to improve system performance or behavior. These include:
+
+* {% ghi 4723 %} {% ghi 5239 %} Changes to properties `tserver.session.idle.max` and
+  `tserver.session.update.idle.max` no longer require a TabletServer restart.
+
+* {% ghi 5078 %} {% ghi 5081} Added property `monitor.extents.obfuscate` (default true) which can be
```

Comment:
This was reverted: see https://github.com/apache/accumulo/pull/5081

Reply:
Thanks @ddanielr . I removed #5081 from the milestone and will remove this.